.github/workflows/unittests.yaml

```diff
@@ -0,0 +1,57 @@
+name: Unittests
+
+on:
+  pull_request:
+
+  push:
+    branches:
+      - main
+      - release_*
+
+jobs:
+  test:
+    name: Run JDK ${{ matrix.jdk-version }} tests
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk-version: [ "19" ]
```

Comment:
better to use 11 because our prod env is using 11 now.

Reply:
added jdk 11